Assets/MixedRealityToolkit.Providers/WindowsVoiceInput/WindowsDictationInputProvider.cs

```diff
@@ -109,7 +120,9 @@ public override void Enable()
         /// <inheritdoc />
         public override void Update()
         {
-            if (!Application.isPlaying || MixedRealityToolkit.InputSystem == null) { return; }
+            IMixedRealityInputSystem inputSystem = Service as IMixedRealityInputSystem;
```

Comment:
This line occurs an awful lot

Reply:
Service is define as an IMixedRealityService to enable it to store any derived type. This line casts to the expected type, in this case IMixedRealityInputSystem in such a way as to get a null, rather than a typecastexception in the event that the wrong type (or null) is stored.